controllers/familyGroupsController.js

```diff
@@ -42,8 +42,10 @@ const createFamilyGroup = async (req, res) => {
 
   try {
     const timeZone = await TimeZone.findById(timeZoneId);
-    const familyGroup = new FamilyGroup({ name, timeZone });
+    if (!timeZone) return res.status(500).send({ message: 'Timezone not exist' });
     const user = await User.findById(userId);
+    if (!user) return res.status(500).send({ message: 'User not exist' });
```

Comment:
lo mismo, status 404, y el mensaje de error "User not found" o "User does not exist"